jax/_src/numpy/lax_numpy.py

```diff
@@ -5954,7 +5954,7 @@ def _multi_slice(arr,
   for starts, limits, removed in safe_zip(start_indices, limit_indices, removed_dims):
     sliced = lax.slice(arr, starts, limits)
     if removed:
-      sliced = sliced.reshape(np.delete(sliced.shape, removed_dims))
```

Comment:
From reading the code, I think this should be using `removed` instead of `removed_dims`, but please let me know if I am missing something.